src/app/pipes/filter-search/customFilter/custom-filter.pipe.ts

```diff
@@ -0,0 +1,20 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'customFilter'
+})
```

Comment:
please check, if we can use the existing pipe only as per title or name based filter

Reply:
Yes vishnu we can use the existing pipe filter, Now code is updated